lib/racker.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+require 'erb'
+require 'yaml'
+require 'codebreaker'
+
+class Racker
+  SCORE_DATABASE = './lib/data/score.yml'
+  GAME_DATABASE = './lib/data/game.yml'
+
+  def self.call(env)
+    new(env).response.finish
+  end
+
+  def initialize(env)
+    @request = Rack::Request.new(env)
+    load_game
+  end
+
+  def response
+    case @request.path
```

Comment:
Move your routes to constant. Using `case` is bad practice.
For example, you can:
```ruby
ROUTES = {
    '/' => ->(instance) { instance.index },
    '/guess' => ->(instance) { instance.make_guess },
    ...
  }.freeze
```
And then using this hash like this:
```ruby
def response
  return ROUTES[@request.path].call if ROUTES[@request.path]
  Rack::Response.new('Not Found', 404)
end
```